llmware/library.py

```diff
@@ -682,7 +1048,21 @@ def add_website (self, url, get_links=True, max_links=5):
 
     def add_wiki(self, topic_list,target_results=10):
 
-        """ Main method to add a wikipedia article to a library - enter a list of topics """
+        """ Main method to add a wikipedia article to a library - enter a list of topics 
```

Comment:
Delete space before you start the doc and add full stop.
```suggestion
        """Main method to add a wikipedia article to a library - enter a list of topics.
```